apps/web/src/components/Messages/RemoteAttachmentPreview.tsx

```diff
@@ -0,0 +1,147 @@
+  sentByMe: boolean;
+}
+
+type Status = 'unloaded' | 'loading' | 'loaded';
+
+const RemoteAttachmentPreview = ({
+  remoteAttachment,
+  profile,
+  sentByMe
+}: RemoteAttachmentPreviewProps): JSX.Element => {
+  const [status, setStatus] = useState<Status>('unloaded');
+  const [attachment, setAttachment] = useState<Attachment | null>(null);
+  const { client } = useXmtpClient();
+  const loadedAttachmentURLs = useAttachmentStore(
+    (state) => state.loadedAttachmentURLs
+  );
+  const addLoadedAttachmentURL = useAttachmentStore(
+    (state) => state.addLoadedAttachmentURL
+  );
+  const cachedAttachments = useAttachmentCacheStore(
+    (state) => state.cachedAttachments
+  );
+  const cacheAttachment = useAttachmentCacheStore(
+    (state) => state.cacheAttachment
+  );
+
+  const redactionReason = useMemo<string | null>(() => {
+    const cached = cachedAttachments.get(remoteAttachment.url);
+
+    // We've already got it, no need to show loading
+    if (cached) {
+      return null;
+    }
+
+    if (profile && !profile.isFollowedByMe && !sentByMe) {
+      return t`Attachments are not loaded automatically from people you don’t follow.`;
+    }
+
+    // if it's bigger than 100 megabytes
+    if (remoteAttachment.contentLength > 104857600 && !sentByMe) {
+      return t`Large attachments are not loaded automatically.`;
+    }
+
+    return null;
+  }, [profile, sentByMe, cachedAttachments, remoteAttachment]);
+
+  const load = useCallback(
+    async function () {
+      const cachedAttachment = cachedAttachments.get(remoteAttachment.url);
+
+      if (cachedAttachment) {
+        setAttachment(cachedAttachment);
+        setStatus('loaded');
+        return;
+      }
+
+      setStatus('loading');
+
+      if (!client) {
+        return;
+      }
+
+      const attachment: Attachment = await RemoteAttachmentCodec.load(
+        remoteAttachment,
+        client
+      );
+
+      setAttachment(attachment);
+      setStatus('loaded');
+
+      cacheAttachment(remoteAttachment.url, attachment);
+      addLoadedAttachmentURL(remoteAttachment.url);
+    },
+    [
+      client,
+      remoteAttachment,
+      addLoadedAttachmentURL,
+      cachedAttachments,
+      cacheAttachment
+    ]
+  );
+
+  useEffect(() => {
+    async function loadInitial() {
+      if (!redactionReason) {
+        await load();
+      }
+    }
+
+    loadInitial();
+  }, [
+    load,
+    client,
+    remoteAttachment,
+    loadedAttachmentURLs,
+    profile,
+    cachedAttachments,
+    redactionReason
+  ]);
+
+  return (
+    <div className="mt-1 space-y-1">
+      {attachment && <AttachmentView attachment={attachment} />}
+      {status === 'loading' && (
+        <Spinner className="mx-28 my-4 h-48 w-48" size="sm" />
+      )}
+      {status === 'unloaded' && (
+        <div className="space-y-2 text-sm">
+          <p>
+            {remoteAttachment.filename} -{' '}
+            {humanFileSize(remoteAttachment.contentLength)}
+          </p>
```

Comment:
I think it's useful to act as a title when someone you are not friends with sends you something:
<img width="529" alt="Screenshot 2023-05-15 at 11 50 46 AM" src="https://github.com/lensterxyz/lenster/assets/556051/0877d4e9-9899-48eb-8464-4331426e5750">

Reply:
Hmm! i suggest we can remove that 🙇🏼 

We can try to avoid as much as tech terms in UI 🙏🏼  @elisealix22

Reply:
Removed. Here's what it will look like now:

<img width="511" alt="Screenshot 2023-05-15 at 1 19 32 PM" src="https://github.com/lensterxyz/lenster/assets/556051/3129d16e-50a1-4e9c-9be8-42de741520f4">